### PR TITLE
Update tox.ini to grab the latest api-stub-generator from the devops artifact feed

### PR DESCRIPTION
**Pull request by tjprescott**
Follow-up to https://github.com/Azure/azure-sdk-tools/pull/3054

### CODE REVIEW

**Review by scbedd — APPROVED** [review]

File: eng/tox/tox.ini [hunk 1]
```diff
@@ -315,7 +315,7 @@ deps =
   {[base]deps}
 commands =
     # install API stub generator
-    {envbindir}/python -m pip install "git+https://github.com/azure/azure-sdk-tools.git@api-stub-generator_v0.2.10#subdirectory=packages/python-packages/api-stub-generator&egg=api-stub-generator"
+    {envbindir}/python -m pip install api-stub-generator --index-url="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/"
```
Comment (scbedd) [inline eng/tox/tox.ini:318]: I am worried about leaving this unpinned. Hotfix release, etc, I don't think `always latest` will be a good experience. 

The retargeted index url definitely should stick around though. Thanks!

Comment (tjprescott) [reply eng/tox/tox.ini:318]: Per our Teams discussion, we'll keep this latest for now.

**Pull request merged**